src/EnumDynamicReturnTypeExtension.php

```diff
@@ -50,11 +50,9 @@ public function __construct()
             return $this->detectGetValueReturnType($class);
         };
 
-        if (method_exists(Enum::class, 'getvalues')) {
-            $this->staticMethods['getvalues'] = function (string $class) {
-                return $this->detectGetValuesReturnType($class);
-            };
-        }
```

Comment:
I'm fine with removing support for php-enum < 4 but this should be reflected in composer.json

Reply:
You're right, I pushed an update